pages/common/fdroid.md

```diff
@@ -1,7 +1,7 @@
 # fdroid
 
 > F-Droid build tool.
-> F-Droid is an installable catalogue of FOSS (Free and Open Source Software) applications for the Android platform.
+> F-Droid is an installable catalog of FOSS (Free and Open Source Software) applications for the Android platform.
```

Comment:
Should US English be preferred over UK English? :innocent: 
https://www.wordreference.com/enfr/catalogue

Reply:
If not, shouldn't we avoid changing English words from UK to US and vice-versa? (it could be endless :blush:)

Reply:
I found the response :innocent: in https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#default-language-for-newly-added-pages (RTFM :wink:):

> The default language used for pages is English (US).

Reply:
Hey there, @nicokosi! Generally speaking we prefer to avoid hiding comments unless absolutely necessary, as it obscures the conversation that has taken place. You can always click the "resolve conversation" button instead. As a side note, it's generally good practice not to hit resolve conversation on important conversations that have occurred on a pull request either, as it makes them more obvious when revisiting later.

Reply:
And to answer your question yeah, we do generally use en_US for English pages. But it's not a completely hard and fast rule - there's no need to go through all our existing pages and fix the language. I'm sure there are some notable exceptions.

Reply:
No problem for not hiding comments (I did it because I answered my own questions 😉).